infra/gravity/ops_center.go

```diff
@@ -0,0 +1,159 @@
+package gravity
```

Comment:
let's name the file `opscenter.go` as we don't use underscores in file names except tests

Reply:
also, can you add license header

Reply:
Re: underscores My bad, there are other files in infra/gravity that have underscores so I was just following what I saw.

Re: license header. Do we have a reference header documented anywhere? It looks like it's missing consistently throughout the project.